### ISSUE TITLE
load via api

### ISSUE THREAD

**Issue by axsweet** [hanchikang/laravel-admin-listbox-sortable#1]
I see same issue on standard packages for load from listbox by api.  any advice? can you modify to load the results on left side by api? or allow once loaded below to go to left side for sorting?

**Comment by hanchikang** [comment]
`$form->listboxsortable($column[, $label])->options('api');`

listboxsortable extends Listbox
Listbox extends MultipleSelect
MultipleSelect extends Select

u can find this on toplink：
[(https://laravel-admin.org/docs/zh/1.x/model-form-fields#Select单选)](https://laravel-admin.org/docs/zh/1.x/model-form-fields#Select单选)

**Comment by axsweet** [comment]
it works fine with multiple select or select 100%. But when I use the sortable listbox, it does not load via Ajax. Any ideas? if I change listboxsortable to mutlipleSelcet boom loads perfectly.  and I use the admin script to populate the right side or the multiple select works the same and awesome but only issue is Ajax doesnt load the options

**Comment by axsweet** [comment]
$form->multipleSelect('heyinterstitialProduct', 'Interstitial Page Products')
            ->options(function ($id) use ($form) {
                if ($form->model()->productProductRelationInterstitialProduct && count($form->model()->productProductRelationInterstitialProduct)) {
                    return $form->model()->productProductRelationInterstitialProduct->pluck('product.name', 'relatedProductID')->toArray();
                }
                return [];
            })->ajax(url('/api/website-items'));

api is tested and loads perfectly on multiple select and select

**Comment by hanchikang** [comment]
usage:
`$form->listboxsortable($column[, $label])->options('/api/options');`

i found a problem and fixed it：incorrect sorting after getting options through API.
please update and run the command:
`php artisan vendor:publish --provider=CKHan\ListboxSortable\ListboxSortableServiceProvider --force`

**Comment by axsweet** [comment]
I am not seeing any data load still. How do I pull in more into the list box via api? 


I will come back to this

**Comment by hanchikang** [comment]
supports loading all options at once throught api. loading more as paging is not suported.
